nginx.tmpl

```diff
@@ -549,7 +571,9 @@ server {
     {{- if $server_tokens }}
     server_tokens {{ $server_tokens }};
     {{- end }}
+    {{- if $http2 }}
     http2 on;
+    {{- end }}
```

Comment:
`http3 on` is default nginx behaviour.
Which seems unlogical for me, because http2 is still disabled by default. I know if you do not set the quic listener it is kind of disabled.
But for reading purposes would it be an idea to add `http3 on`? I think for debugging purposes it is more clear if you see that directive instead of assuming. And maybe the default would change if nginx thinks it's more mature.

Reply:
Agreed, and thanks for clarifying that it is on by default, I was under the impression that `http3 on` did nothing. But why on earth did they made it default to on if it's considered an experimental feature ?

Reply:
Done ✅